src/components/Patient/PatientInfoCard.tsx

```diff
@@ -59,6 +63,16 @@ export default function PatientInfoCard(props: PatientInfoCardProps) {
   const subpathMatch = usePathParams("/facility/:facilityId/*");
   const facilityIdExists = !!subpathMatch?.facilityId;
   const { t } = useTranslation();
+  const user = useAuthUser();
+  const permissions = getPermissions(
+    (perm, perms) => perms.includes(perm),
+    user?.permissions || [],
+  );
+  const canViewPatient = permissions.canViewPatients;
+  const isAdmin =
+    user.user_type === "administrator" || user.is_superuser === true;
```

Comment:
let's not check against user type. this does not define the role of the user on that facility organization. 

rely only on permissions to check if user has access to do something.